models/klaviyo__events.sql

```diff
@@ -0,0 +1,116 @@
+{{
+    config(
+        materialized='incremental',
+        unique_key='event_id',
+        partition_by={
+            "field": "occurred_on",
+            "data_type": "date"
```

Comment:
Curious, does this work on spark? I know there were issues with `int_klaviyo__event_attribution` incremental logic on spark

Reply:
yeah so spark requires the partition_by attribute to be a list, [this](https://github.com/fivetran/dbt_klaviyo/blob/spark_compatibility/models/intermediate/int_klaviyo__event_attribution.sql#L8) somewhat worked (though the model run hangs on circle)